server/statistics/hot_peer_cache_test.go

```diff
@@ -392,29 +408,28 @@ func (t *testHotPeerCache) testMetrics(c *C, interval, byteRate, expectThreshold
 }
 
 func (t *testHotPeerCache) TestRemoveFromCache(c *C) {
+	peerCount := 3
+	interval := uint64(5)
 	checkers := []check{checkAndUpdate, checkAndUpdateWithOrdering}
 	for _, checker := range checkers {
 		cache := NewHotPeerCache(Write)
-		region := buildRegion(Write, 3, 5)
+		region := buildRegion(Write, peerCount, interval)
 		// prepare
+		intervalSums := make(map[uint64]int)
 		for i := 1; i <= 200; i++ {
-			checker(c, cache, region)
+			rets := checker(c, cache, region)
+			checkIntervalSumContinuous(c, intervalSums, rets, interval)
```

Comment:
```suggestion
			checkIntervalSumContinuous(c, intervalSums, checker(c, cache, region), interval)
```

Reply:
it will be too long and difficult to understand.